install/install.php

```diff
@@ -81,9 +81,6 @@ function initialize_inputs()
 	if (ini_get('magic_quotes_sybase') || strtolower(ini_get('magic_quotes_sybase')) == 'on')
```

Comment:
kill this too

Reply:
I think I left it as a warning, but... yeah, doesn't make much sense I guess.
Removed.